pkg/generate/podtemplatespec.go

```diff
@@ -225,7 +228,7 @@ func PodTemplateSpec(cr *v1alpha1.ImageRegistry, p *parameters.Globals) (corev1.
 			Containers: []corev1.Container{
 				{
 					Name:  "registry",
-					Image: cr.Spec.ImagePullSpec,
```

Comment:
@bparees What is `ImagePullSpec` ? Should we ignore this field? If so, why is it needed in `OperatorSpec` ?

Reply:
@bparees The operator api has `ImagePullPolicy`. Since we don't use `ImagePullSpec`, should we use `ImagePullPolicy` from the custom resource or we will get it from another magic env variable ? 

https://github.com/openshift/api/blob/master/operator/v1alpha1/types.go#L31